core/auth/keycloak/add-tenant.py

```
@@ -0,0 +1,512 @@
+# --------------------------------------------------------------------------- #
+
+
+def http(method, url, token=None, data=None, content_type="application/json"):
+    body = None
+    if data is not None:
+        if content_type == "application/x-www-form-urlencoded":
+            body = urllib.parse.urlencode(data).encode()
+        else:
+            body = json.dumps(data).encode()
+
+    req = urllib.request.Request(url, data=body, method=method)
```

Ruff (0.15.12)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)